src/transformers/generation_logits_process.py

```diff
@@ -702,3 +702,33 @@ class LogitNormalization(LogitsProcessor, LogitsWarper):
     def __call__(self, input_ids: torch.Tensor, scores: torch.Tensor) -> torch.Tensor:
         scores = scores.log_softmax(dim=-1)
         return scores
+
+
+class SuppressTokensAtBeginLogitsProcessor(LogitsProcessor):
```

Comment:
We could transform this also into a logit processor that forces the 2nd, 3rd and 4th token

Reply:
On it!

Reply:
We can delete this logits processor no? It shouldn't be used anymore

Reply:
I am still using it, as it prevent the sampling of " " (blank) after the forced_decoder_ids